.github/config/.golangci.yml

```diff
@@ -8,4 +8,4 @@ output:
 issues:
```

Comment:
Not sure if this comment is within the scope of this PR, but ...

This seems like extremely permissive lint settings.  We should at VERY least be turning on gosec.  You might consider using this config:
https://gist.github.com/maratori/47a4d00457a92aa426dbd48a18776322